### ISSUE TITLE
Website domain

### ISSUE THREAD

**Issue by iambeingtracked** [sainnhe/everforest#73]
I've been thinking about everforest having a domain name like [Nord](https://www.nordtheme.com/) or [Dracula](https://draculatheme.com/) has. It would be really nice to have a website to present people how the color scheme looks. It would also help the theme gain popularity. For the domain itself i think everfo.rest or everfore.st would fit the best. There's also everforest.agency, but I think it would fit better for a fan-made website.

**Comment by sainnhe** [comment]
Sounds interesting, but I'm not a web developer so I can't create a beautiful website currently.

**Comment by iambeingtracked** [comment]
Me neither. But this can be made as a community project. It is possible to host the website's source on git(hub) and make it automatically sync with the changes in the repo. This way, someone can make a push request with their changes and if it gets merged to upstream, the website synchronizes with the repository.

**Comment by antoineco** [comment]
The Nord website is great because it contains a formal description of the color palette, with the intended usage of each color in that palette. In that regard it is made first and foremost for designers. It reminds me of the website for the Solarized color scheme.

The Everforest palette was created more empirically, so to make a web page interesting we probably first need a good formal description of how the colors work together with each contrast setting.

Also both Nord and Dracula have a lot of official ports. Everforest only has Vim and VSCode. To have some contents to show on a web page, we would need a few solid community maintained ports. The wiki lists a few.

**Comment by iambeingtracked** [comment]
> oremo

Yes, that is true. Everforest currently lacks ported themes, we can wait till more get created, Also, I think it would be nice if there was actually some kind of a standard for how colors are used, because now people just use the given colors however they want. It is good, because it allows for more creativity while still having the ability to have your own place in the wiki, but can create some unfortunate moments with porting. I think we'll need to wait.

**Comment by kmkinney** [comment]
I have some experience with web design and have loved the Everforest theme. I use it for everything! If at some point this becomes a thing I'd love to help.

**Comment by sainnhe** [comment]
I want to share some my ideas on "community" and "official".

When I first created gruvbox-material I thought about creating a github organization to maintain it, so it will become more maintainable and can gain more attention.

I shared my idea with another maintainer of gruvbox-material, but he refused, he said he prefer a more open approach to development, like gruvbox.

There is no official organization of gruvbox, thus no such things like "official ports", everything but [morhetz/gruvbox](https://github.com/morhetz/gruvbox) is third-party. And there is a place to gather information of some ports of gruvbox for other apps: [morhetz/gruvbox-contrib](https://github.com/morhetz/gruvbox-contrib), if one wants to share his port, he can fork this repository and open a PR.

I found that I actually like this idea, it reminded me of the speech on Git by Linus Torvalds, he thought the correct way to develop open source projects should be distributed and decentralized.

So I decided to use a more open development mode:

- No organization, thus no official ports.
- If you want to port this color scheme to another app, go ahead and do it, don't worry about being "non-official".
- If you think the existing port of an app is bad, go ahead and create your own, there is no need to worry about the existing port is "official" while your port is not.
- If you want to share your port, you can directly edit the wiki page, no need to open a PR and wait for the maintainers to merge it.
- It's also fine if you don't want to add your project to the wiki page, all depend on you.
- If you think this vim color scheme needs improvement, you can open a PR, or fork this repository and maintain your own.
- You can also create a lua based color scheme for neovim if you don't like this vim script implementation.
- The only special point of this repository is that the color palette is originally designed here.

Of course this development mode has some downsides, one of which is that this repository may lose maintaince one day.

But unlike other types of open source projects, the essence of color schemes is the color palette, not the implementation. So even if this repository loses maintaince one day, the everforest color scheme itself will not lose maintaince, because the design of the color palette has already finished.

Gruvbox hasn't been updated in two years, but it's still one of the most famous color schemes out there, and nowadays people still actively create new gruvbox based projects.

And I don't think this repository will easily lose maintaince. @jef and @antoineco did a very great job and we'll do our best to keep maintaining it.

---

I understand people like this color scheme and want to contribute for it, sincere thanks for you. But I think instead of creating an organization and maintaining "official" ports, a better approach is simply sharing it to your friends, creating your own ports or opening PRs to help us improve the code.

As for the website, my original idea is not making it as a home page like Nord or Dracula, I just want a place to demonstrate the color palette, and maybe add a link of the wiki page to tell people where to find the ports. The source code may be hosted in a different branch of this repository and the website can be published via github pages. Or if you think it's inconvenient to maintain, you can also create your own repository to host the source code.

**Comment by iambeingtracked** [comment]
I fully agree.

**Comment by antoineco** [comment]
Looks like @jamesalexatkin started something a while ago :)

https://jamesatk.in/everforest-web/

**Comment by sainnhe** [comment]
It looks beautiful!

**Comment by iambeingtracked** [comment]
Looks pretty bare, but it's still awesome

**Comment by jef** [comment]
as much as i agree with @sainnhe, i think it would be worth looking into github pages and creating a long living branch (gh-pages) that hosts a page via https://sainnhe.github.io/everforest.

just wanted to throw that one out there 😛

**Comment by jamesalexatkin** [comment]
Hey just seen this 😄 I hadn't heard that much about open source philosophy before but there is something very elegant about what you say @sainnhe!

With regard to the site I was making, I was messing around with React and trying to put something together. There is more than what renders on the link but am trying to solve an issue with `BrowserRouter` not working with Github Pages.

The plan though has been to host links to the current ports people have made (and keep it updated of course), as well as the palettes that make up the colour scheme

**Comment by antoineco** [comment]
Not a static page per-se, but I started documenting the palette and usages of each color at https://github.com/antoineco/sainnhe-everforest/blob/palette/palette.md

There is a PR open at #92 to add this document to the repo (still a draft).

**Comment by iambeingtracked** [comment]
> Not a static page per-se, but I started documenting the palette and usages of each color at https://github.com/antoineco/sainnhe-everforest/blob/palette/palette.md
> 
> There is a PR open at #92 to add this document to the repo (still a draft).

I think #2b3339 is missing there

**Comment by mtergel** [comment]
I love this color scheme and would like to help out building out the web

**Comment by Theory-of-Everything** [comment]
Hey, if you do need somebody knowledgeable with static webdev, I can pitch in a bit! (for reference, you can see my own website)

**Comment by antoineco** [comment]
@Theory-of-Everything if you feel like working on some static page for Everforest, with a simple overview of the palette and existing ports (maybe a screenshot for each?) you're welcome to do so! We'll link the repo to it.

**Comment by jamesalexatkin** [comment]
Managed to solve my problem with the React router 😄 I plan to add palettes next, but this site has it working with the apps for now: https://jamesatk.in/everforest-web/#/

**Comment by iambeingtracked** [comment]
That looks awesome! Could you change the links to some of the themes I made? I moved all of the themes to sourcehut and that results to some links giving a 404

**Comment by jamesalexatkin** [comment]
@iambeingtracked Yeah of course 🙂 Have you got the links you'd like to add/replace?

**Comment by iambeingtracked** [comment]
@jamesalexatkin, WezTerm should be https://git.sr.ht/~maksim/wezterm-everforest
Element should be https://git.sr.ht/~maksim/element-everforest
Zathura should be https://git.sr.ht/~maksim/zathura-everforest
Thank you

**Comment by jamesalexatkin** [comment]
Hi all! After a couple of years inactive on this I decided to have another crack at it, this time as a static site. I think React was a bit overkill for it at the time and I just motivation to be honest.

It's now hosted using GitHub Pages at: https://github.com/jamesalexatkin/everforest-homepage

Have a look and see what you think! Open to any feedback and of course adding anything I might have missed 😄

**Comment by antoineco** [comment]
I love the idea of the drop-down and the code box!

A few suggestions for improvement in case you are willing to invest more time into this:
- Some of the logos appear really big on Safari.
- The highlight semantics in the code box do not actually match those of Everforst.
- The text while hoevering the palette colors appears with a delay.

**Comment by jamesalexatkin** [comment]
@antoineco Thanks for the feedback 😊 

* Which logos are looking too big on Safari?
* I'll have to look into this, Prism doesn't actually look like it's been updated in a while so maybe another library would be better here.
* I've fixed the tooltip hover delay now on the palettes.

**Comment by Theory-of-Everything** [comment]
> Hi all! After a couple of years inactive on this I decided to have another crack at it, this time as a static site. I think React was a bit overkill for it at the time and I just lost motivation to be honest.
> 
> It's now hosted using GitHub Pages at: http://jamesatk.in/everforest-homepage [(repository)](https://github.com/jamesalexatkin/everforest-homepage)
> 
> Have a look and see what you think! Open to any feedback and of course adding anything I might have missed 😄

Nice work! I had a page on my website (https://theoryware.net/everforest) but it was mostly palette reference for the trillions of things I use it for.

One suggestion: It would be really cool to see screenshots of the particular editor plugins with some source file for a generalist preview of how the scheme looks
